app/assets/stylesheets/mobile/mobile.scss

```diff
@@ -27,6 +27,11 @@ body {
   padding: 0px;
 }
 
+html, body, #app {
+    height: 100%;
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Each selector in a comma sequence should be on its own line